src/compiler/interface.jl

```diff
@@ -28,7 +28,37 @@ end
 
 # Wrappers
 
-_pullback(f, args...) = _pullback(Context(), f, args...)
+function _pullback(f, args...)
```

Comment:
This is the wrong function to overload as it will not be hit for functions that occur in functions Zygote decomposes